ClrMD.Extensions/ClrMDExtensions.cs

```diff
@@ -134,10 +140,18 @@ public static List<StackFrameInfo> GetDetailedStackTrace(this ClrThread thread)
         public static string GetStackTrace(this ClrThread thread)
         {
             StringBuilder builder = new StringBuilder();
+            int count = 0;
+            foreach (ClrStackFrame frame in thread.EnumerateStackTrace())
+            {
 
-            foreach (ClrStackFrame frame in thread.StackTrace)
-                builder.AppendLine(frame.DisplayString);
+                if (frame != null)
+                    builder.AppendLine($"{frame}");
```

Comment:
Can frame really be null here? That would be unexpected.

This should not use a string interpolation, a single `builder.AppendLine(frame.ToString()` will be more efficient.

Is frame.ToString() similar to the previous frame.DisplayString?